packages/appstore/src/components/ready-to-update-wallets/upgrade-info-list.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TReadyToUpgradeForm } from './ready-to-upgrade-form';
+import { isMobile } from '@deriv/shared';
+
+type TUpgradeInformationList = {
+    name: string;
+    visiblity: boolean;
+    content: React.ReactElement;
+};
+
+const UpgradeInformationList = ({ is_high_risk, is_eu }: TReadyToUpgradeForm) => {
+    const list: TUpgradeInformationList[] = [
+        {
+            name: 'upgrade_info',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text='During the upgrade, deposits, withdrawals, transfers, and adding new accounts will be unavailable.' />
+            ),
+        },
+        {
+            name: 'open_positions',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text="Your open positions won't be affected and you can continue trading." />
+            ),
+        },
+        {
+            name: 'deriv_p2p',
+            visiblity: !is_eu,
```

Comment:
check the behaviour for  `low-risk/eu`, it's not expected.
![image](https://user-images.githubusercontent.com/82078941/231665761-0b705f74-33d0-4dc1-a547-ba70295dee10.png)